scripts/CEE/drain-reboot-nodes/script.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+#
+# Drain and Reboot a worker node.
+# Input:
+#  (Env) WORKER
+#    name of the worker node to be drained / rebooted
+
+set -e
+set -o nounset
+set -o pipefail
+
+CURRENTDATE=`date +"%Y-%m-%d %T"`
+
+if [[ -z "${WORKER}" ]]; then
+    echo "[Error] Variable WORKER cannot be blank."
+    exit 1
+fi
+
+start_job(){
+    echo "Job started at $CURRENTDATE"
+    echo ".................................."
+    echo
+}
+
+finish_job(){
+    echo
+    echo ".................................."
+    echo "Job finished at $CURRENTDATE"
+}
+
+## Function which checks if the node is a Worker node
+check_worker(){
+
+    echo "Checking if \"${WORKER}\" is a worker node..."
+    
+    if (oc get nodes -l node-role.kubernetes.io/worker=,node-role.kubernetes.io/infra!= -oname | grep "${WORKER}") &> /dev/null; then
+        echo "[OK] \"${WORKER}\" is a worker node. Proceeding with draining..."
+        echo
+    else
+        echo "[Error] \"${WORKER}\" is not a worker node. Exiting script..."
+        exit 1
```

Comment:
`exit 1` will call the job failed, but as it is exiting as expected, maybe we do not exit as error?

Reply:
I think `exit 1` makes sense here, as we want to mark the job as failed, and the user can check the logs to know the error.